design/invitation.go

```diff
@@ -0,0 +1,59 @@
+package design
+
+import (
+	d "github.com/goadesign/goa/design"
+	a "github.com/goadesign/goa/design/apidsl"
+)
+
+var _ = a.Resource("invitation", func() {
+
+	a.BasePath("/invitations")
+
+	a.Action("createGroupInvite", func() {
+		a.Security("jwt")
+		a.Routing(
+			a.POST("group/:inviteTo"),
```

Comment:
Why POST `/invitations/group/:id` ? Why not just POST `/invitations/:id`?
/group/ is a bit confusing here IMO.

Reply:
URL has been updated to support both invitation types now.